### ISSUE TITLE
Error: Download removed from Dissenter.com site

### ISSUE THREAD

**Issue by tcreek** [gab-ai-inc/gab-dissenter-extension#111]
What;s going on with that?

I have ZERO interest on any browser based on Google's Chrome.

**Comment by jakubgs** [comment]
Yeah. I don't care about a browser, give me the extension download link.

**Comment by endolith** [comment]
Duplicate of https://github.com/gab-ai-inc/gab-dissenter-extension/issues/106

**Comment by VidarrKerr** [comment]
I have both the chrome and the firefox extensions saved if you want them. I also complained about this browser being the only option. They should put the latest extensions available for download on the Dissenter site.

**Comment by tcreek** [comment]
They have them here already on GitHub. The point is they removed them off their site, have not done any more work on the extensions, and instead pushing that horrible Goolag based garbage Chromium. We do not need to be giving Goolag more control over the internet.

**Comment by VidarrKerr** [comment]
Ah, I did not look here to find them because I have the latest ones before they stopped updating. Thanks for the info.

I think it was a HUGE MISTAKE to stop making the extensions. As a successful entrepreneur, I wish I could get someone at GAB to respond back to me about the development of their products. They are making many grave mistakes. 

There are many courses of actions that could easily be taken to make Dissenter explode on the world scene. It almost seems like they don't want that to happen. Anyone else interested, feel free to contact me here or vidarrkerr at gmail.

**Comment by jazzerciseme** [comment]
Prefer more control over browser. I use https://chromium.woolyss.com/ and https://github.com/NeverDecaf/chromium-web-store for plugins. This plugin definitely needs a come back!

**Comment by ThePharaohArt** [comment]
> They have them here already on GitHub. The point is they removed them off their site, have not done any more work on the extensions, and instead pushing that horrible Goolag based garbage Chromium. We do not need to be giving Goolag more control over the internet.

fyi Dissenter browser is based on Brave not Google's chrome

**Comment by tcreek** [comment]
> > They have them here already on GitHub. The point is they removed them off their site, have not done any more work on the extensions, and instead pushing that horrible Goolag based garbage Chromium. We do not need to be giving Goolag more control over the internet.
> 
> fyi Dissenter browser is based on Brave not Google's chrome

Sorry to tell you, but Brave is forked Chromium.

**Comment by andrebu** [comment]
Is Chromium=Goolag?  Isn't Chromium the defanged open source browser onto which Goolag tacks on all their malware?  

Meaning, Chromium, Brave and Dissenter are fine, while Googlag's Chrome is not, no?

**Comment by tcreek** [comment]
> Is Chromium=Goolag? Isn't Chromium the defanged open source browser onto which Goolag tacks on all their malware?
> 
> Meaning, Chromium, Brave and Dissenter are fine, while Googlag's Chrome is not, no?

That is correct. With Goolag gaining more control over the web (~93% market share?), people need to stop using Chromium based browsers.

**Comment by andrebu** [comment]
Don't mean to be obtuse, but want to clarify this: 

1. If Chromium is made by goolag, does that mean Chromium has goolag's spyware, ad-bloat and malware built in?  
2. Pretty much the same question but for Dissenter browser and Brave -- since they're built on Chromium, does that mean they have googlag's malware built in, or do they remove it before adding their features? 

I was under the impression that they clean Chromium (if anything to clean) before adding their features and releasing.

**Comment by tcreek** [comment]
Chromium only has open source stuff in it. Not sure exactly what it contains, but Brave is a fork of Chromium with their own telemetry stuff in it. Dissenter is a fork of Brave.

No matter what you use above, even Chrome, they all get counted as Chromium when you hit web sites. Why would why developers make sure other browser are compatible with all browsers, and add-ons if well over 90% of people are Chromium based browsers?

**Comment by tcreek** [comment]
Just to make a point of example. I visited Fox News with the latest Firefox browser, and their commenting section no long functions with Firefox. A while back I used to at least get a warning with Waterfox Classic that the browser is no longer supported, and would not show the comments. Now the comment section will not even load with current Firefox.
